keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -1114,6 +1115,11 @@ We plan to add e2e test under api-machinery for crd expression validation:
 - Optimization on super-linear complexity growth
 - Adding metric of the latency of CEL evaluation for CRD evaluation
 
+#### GA
+
+- Formatted message
+- Scalability evaluation
```

Comment:
Can you be a bit more explicit with this? What exactly you're going to evaluate?

Reply:
Sure. We used to have [scale target for CRD](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/95-custom-resource-definitions/README.md#scale-targets-for-ga). And this is targeting to do the similar evaluation and make sure CEL would not add much on scalability side. Will update the session with more details

Reply:
A principled way to handle this might be define a set of CEL rules on a CRD that has a "CEL runtime cost" that is just below our runtime cost max limit. We could then evaluate our pre-existing scalability targets this CRD.